fabtests/pytest/pytest.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+markers =
+    unit : unit tests runs on single instance
```

Comment:
runs --> run? or maybe 'single instance unit tests'?

Reply:
Thank you! Fixed in new revision.